.github/workflows/release-test.yaml

```diff
@@ -0,0 +1,77 @@
+name: Release-Test
```

Comment:
Can we make the name a bit less generic? E.g., `Debian packages` or something? (I guess "Test" is optional since obvious).

This applies also to the workflow file name

Reply:
oh i was gonna add rpm package tests to the same workflow and wanted it to trigger when pushing to release_test and when tagging branches